pkg/certs/cert-inspection/certgraphanalysis/metadata_options.go

```diff
@@ -149,6 +149,15 @@ var (
 			return timestampReg.ReplaceAllString(path, "<timestamp>.pem")
 		},
 	}
+	RewritePrimaryCertBundleSecret = &metadataOptions{
+		rewriteSecretFn: func(secret *corev1.Secret) {
+			if secret.Namespace != "openshift-ingress" || !strings.HasSuffix(secret.Name, "-primary-cert-bundle-secret") {
+				return
+			}
+			hash := strings.TrimSuffix(secret.Name, "-primary-cert-bundle-secret")
+			secret.Name = strings.ReplaceAll(secret.Name, hash, "<hash>")
```

Comment:
this ends up: `<hash>-primary-cert-bundle-secret` how does it help the TLS registry to track this secret?

Reply:
ah, we expect a stable name, don't we ?

Reply:
Yes, a stable name across different runs is a requirement (similar to etcd serving certs we store as `etcd-<master-1>-serving` and so on)

Reply:
ok, sgtm.